src/app/templates/TabBar.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react';
+
+import clsx from 'clsx';
+
+import { TestIDProps } from 'lib/analytics';
+import { T, TID } from 'lib/i18n';
+import { Link } from 'lib/woozie';
+
+interface Props {
+  activeTabName: string;
+  tabs: TabInterface[];
+  withOutline?: boolean;
+}
+
+interface TabInterface extends TestIDProps {
+  name: string;
+  titleI18nKey: TID;
+}
+
+export const TabsBar = React.forwardRef<HTMLDivElement, Props>(({ activeTabName, tabs, withOutline }, ref) => (
```

Comment:
the same, I think it's a little bit confusing that 2 components based on one file

Reply:
No matter how many exports have the file. when we write a lot of components and a lot of interfaces it makes the file too long and less understandable and readable, so I think we have to split them into small codes